rabbitx/client/endpoints/order.py

```diff
@@ -13,8 +13,8 @@ class OrderType(enum.Enum):
 
     MARKET = 'market'
     LIMIT = 'limit'
-    STOP_LOSS = 'stop_loss'
-    TAKE_PROFIT = 'take_profit'
+    STOP_LOSS = 'stop_loss_limit'

```

Comment:
please change the naming to STOP_LOSS_LIMIT and TAKE_PROFIT_LIMIT, add additional STOP_LOSS = 'stop_loss" and TAKE_PROFIT = 'take_profit' for market order stop loss / take profit